raft/src/test/java/org/apache/kafka/raft/RaftClientTestContext.java

```diff
@@ -1818,9 +1839,9 @@ private short fetchSnapshotRpcVersion() {
         }
     }
 
-    private short voteRpcVersion() {
+    short voteRpcVersion() {
         if (kip853Rpc) {
-            return 1;
+            return ApiKeys.VOTE.latestVersion();
```

Comment:
I see. Are you planning to fix/change this in a future PR?

Reply:
yes, discussed offline